lib/src/main/java/nl/knaw/dans/lib/dataverse/DatasetApi.java

```diff
@@ -29,10 +31,12 @@
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 public class DatasetApi extends AbstractApi {
 
     private static final Logger log = LoggerFactory.getLogger(DatasetApi.class);
+    private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
This library uses one ObjectMapper, namely the one created in DataverseClient (or provided to it by the code that instantiates DataverseClient). This way as a user fo the library you can easily see what ObjectMapper features are active.

So, instead of creating an extra ObjectMapper here, delegate the object mapping to `HttpClientWrapper`.